### DISCUSSION TITLE
Camera switch issue on iPhone while uploading ID

### DISCUSSION THREAD

**Discussion by istieak-a** [community/community#80804] [GitHub Education] [answered]
### Hi everyone,

Basically I'm facing an issue when I try to upload my ID. I am using iPhone and I can't switch camera it stucks. I can see the back camera option but it doesn't switch. Can anyone help? I have tried multiple browsers!!

**Comment by manik-007** [comment]
Hello @istieak-a  

You can click a picture of your id with your iphone and upload it via laptop. In this way you can apply for the developer pack.

I hope this helps you. If you'll face any doubt then don't hesitate to ask. I'll be there to help :)

**Reply by istieak-a** [reply]
Sorry but ln desktop there's no option to upload they force to use the webcam:)

**Comment by manik-007** [comment, marked as answer]
@istieak-a Kindly check it again. I have applied for github developer pack 1 week back for two of my friends and i had uploaded the image there.
Don't use github desktop. Apply from gtihub website using any browser.
